### PR TITLE
update doc -- segementation fault referring to FAQ

### PR DESCRIPTION
**Pull request by Crysple**

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### Cannot get trials' metrics in OpenPAI mode
+In OpenPAI training mode, we start a rest server which listens on 51189 port in NNIManager to receive metrcis reported from trials running in OpenPAI cluster. If you didn't see any metrics from WebUI in OpenPAI mode, check your machine where NNIManager runs on to make sure 51189 port is turned on in the firewall rule. 
+
+### Segmentation Fault (core dumped) when installing from source code
+> make: *** [install-XXX] Segmentation fault (core dumped)
+There are two options:
+
+* Update or reinstall you current python's pip like `python3 -m pip install -U pip`
+* Install nni with --no-cache-dir flah like `python3 -m pip install nni --no-cache-dir`
```
Comment (yds05) [inline docs/FAQ.md]: typo, 'flah' -> 'flag'

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+This page is for frequent asked questions and answers.
+
+
+### tmp folder fulled
+NNICTL will use tmp folder as a temporary folder to copy files under codeDir when executing experimentation creation. 
+When met errors like below, try clean up **tmp** folder first.
```
Comment (xuehui1991) [inline docs/FAQ.md]: try to clean up...?

File: docs/FAQ.md (hunk 1 above)
Comment (Crysple) [reply docs/FAQ.md]: has fixed it

File: docs/FAQ.md (hunk 2 above)
Comment (Crysple) [reply docs/FAQ.md]: thanks, will modify.
Sorry for not checking this file since it is directly copied from WIKI

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+> OSError: [Errno 28] No space left on device
+
+### Cannot get trials' metrics in OpenPAI mode
+In OpenPAI training mode, we start a rest server which listens on 51189 port in NNIManager to receive metrcis reported from trials running in OpenPAI cluster. If you didn't see any metrics from WebUI in OpenPAI mode, check your machine where NNIManager runs on to make sure 51189 port is turned on in the firewall rule.
```
Comment (xuehui1991) [inline docs/FAQ.md]: "nni_manager" and "NNIManager" are both in this file. I think you could choose one.

Comment (Crysple) [reply docs/FAQ.md]: ok, will fix

**Pull request merged**